backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -4693,6 +4693,61 @@ template <typename Torus> struct int_div_rem_memory {
   }
 };
 
+template <typename Torus> struct int_scalar_mul_high {
+  int_radix_params params;
+  bool allocate_gpu_memory;
+
+  int_fullprop_buffer<Torus> *fullprop_mem;
```

Comment:
You can now remove this since it won't be used in the host function :slightly_smiling_face: